pom.xml

```diff
@@ -28,16 +28,16 @@ THE SOFTWARE.
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.29</version>
+        <version>2.30</version>
     </parent>
 
     <artifactId>collapsing-console-sections</artifactId>
     <version>1.6.1-SNAPSHOT</version>
     <packaging>hpi</packaging>
     
     <properties>
-        <jenkins.version>1.609.3</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>2.46</jenkins.version>
+        <java.level>8</java.level>
```

Comment:
It's not going to work in such way.
Firstly, there is n reason to update the LTS dependency. Then, 2.46.x requires compatibility with Java 7.
Please revert these changes or explain why do you need them